modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -973,70 +973,86 @@ definition {
 
 		var slaName = "SLAName";
 
-		WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
+		task ("Given a blog entry with an SLA and the account language is set to English") {
+			WorkflowMetrics.executeIndexActions(actionDescription = "Workflow Indexes");
 
-		Click(locator1 = "WorkflowMetricsSLA#ADD_NEW_SLA_BUTTON");
+			WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
 
-		WorkflowMetrics.fillSLAForm(
-			slaDefinitionDays = 1,
-			slaDefinitionDescription = "SLA Description",
-			slaDefinitionEnd = "Process Ends: Approved",
-			slaDefinitionHours = 0000,
-			slaDefinitionName = ${slaName},
-			slaDefinitionStart = "Process Begins");
+			Click(locator1 = "WorkflowMetricsSLA#ADD_NEW_SLA_BUTTON");
```

Comment:
I've added a workflow reindex to make sure that there is no duplicate Single Approver Processes in Metrics.

Reply:
I've also added tasks to provide better readability of this test.